.github/workflows/rust.yml

```diff
@@ -11,15 +11,45 @@ env:
 
 jobs:
   test_linux:
-    name: Test Linux
+    name: Test Linux Default
     runs-on: ubuntu-latest    
     steps:
     - uses: actions/checkout@v2
     - name: install dependencies
       run:  |
-        sudo apt install coinor-libcbc-dev coinor-cbc
+        sudo apt install coinor-cbc
     - name: Build
-      run: cargo build --verbose
+      run: cargo test --no-run --verbose
```

Comment:
For me, it is more clear to trigger `cargo build` here, since if the tests don't compile later we know that at least the problem is not in the crate itself. Just a matter of taste, it doesn't matter too much.